docker/Dockerfile

```diff
@@ -10,10 +10,14 @@ RUN apt-get update &&  apt-get install -y --no-install-recommends \
 # Without this, we are unable to pick up more recent packages
 COPY docker/Rprofile.site /usr/local/lib/R/etc/Rprofile.site
 
-COPY DESCRIPTION /tmp/DESCRIPTION
```

Comment:
These changes unrelated, but avoid hitting GH rate limits when building the image, following the pattern we use now in other orderly images